installing/installing_azure_stack_hub/manually-creating-iam-azure-stack-hub.adoc

```diff
@@ -22,11 +22,13 @@ include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[level
 
```

Comment:
This Manually creating IAM assembly is no longer in the TOC, so just want to verify that you aren't missing any changes you made here in the output :)

Reply:
By design. Given that ASH on support CCO in manual mode, I decided to only use `modules/manually-create-identity-access-management.adoc`